lib/charms/observability_libs/v0/metrics_endpoint_discovery.py

```diff
@@ -47,7 +47,7 @@ def _on_endpoints_change(self, event):
 from pathlib import Path
 from typing import Dict, Iterable
 
-from lightkube import Client
+from lightkube import Client  # pyright: ignore
```

Comment:
Iirc, if we import `from lightkube.client` we wouldn't need the ignore.

Reply:
Yes and no, this ends up being a lot easier; as a reminder, `mypy` was happily skipping this until now :)